Fix some typos for Episode 1 and 2

Missake212 opened the pull request:
Fix two typos (noticed more but I'm not sure how to fix them yet so I won't touch them for now).

<img width="3840" height="2160" alt="Screenshot 2025-10-17 175316" src="https://github.com/user-attachments/assets/08911578-4379-4c5c-9101-8dda8c70b883" />

In the Japanese text there's no space and since it's a laugh I don't think there should be one.

<img width="3840" height="2160" alt="Screenshot 2025-10-19 033338" src="https://github.com/user-attachments/assets/6923f93c-81cf-4ff5-af25-fbfb433f612e" />

Missake212 committed:
fix typos

Missake212 committed:
Add files via upload

Missake212 committed:
fix grammatical error

Missake212 committed:
missing "the"

Missake212 committed:
another missing "the"

Missake212 committed:
Missing "."

Missake212 committed:
delete extra ","

Missake212 committed:
missing "to"

Missake212 committed:
another missing "to"

Missake212 committed:
fix typo

Missake212 commented:
@drojf Sure thing, I'm close to the end of Episode 2 right now, once I'm done I'll ping you here or on the Discord, whichever you prefer. Next time I'll keep the file on my PC and make changes locally instead of committing every single one, thought I would just fix 2-3 typos but found way more as I've been reading so I kinda haven't stopped making small changes as I've been reading more.

drojf commented:
> Next time I'll keep the file on my PC and make changes locally instead of committing every single one

TBH you can do whatever is convenient (and I guess also as a "backup" of your changes in case your PC dies), since I can always merge the commits into one at the end.

Missake212 committed:
change "Beatrice" to "Beato"

Missake212 committed:
replace extra "and" with a ","

Missake212 committed:
fix typo

Missake212 committed:
replace "and" by ","

Missake212 committed:
change "giving" to "give"

Missake212 committed:
typo fix ep 3

Changing "letting you to succeed when you shouldn't have" to "letting you succeed when you shouldn't have"

Missake212 commented:
Last thing I'll touch since I'm not playing on a computer anymore, changing "letting you to succeed when you shouldn't have" to "letting you succeed when you shouldn't have", could've also changed it to "leading you to succeed when you shouldn't have" not sure which of the two is the better option.

<img width="3840" height="2160" alt="Capture d&#39;écran 2025-12-14 192642" src="https://github.com/user-attachments/assets/a5c6b3f3-28ec-4741-bc72-9f45ffec7e6a" />
